crates/requirements-txt/src/lib.rs

```diff
@@ -581,7 +581,6 @@ fn parse_entry(
     Ok(Some(if s.eat_if("-r") || s.eat_if("--requirement") {
         let requirements_file = parse_value(content, s, |c: char| !['\n', '\r', '#'].contains(&c))?;
         let end = s.cursor();
-        eat_trailing_line(content, s)?;
```

Comment:
Not necessary, I think, because the next step after parsing an entry is to eat the trailing line.